les/fetcher.go

```diff
@@ -674,6 +829,10 @@ func (f *lightFetcher) checkKnownNode(p *peer, n *fetcherTreeNode) bool {
 	return n.known
 }
 
+func (f *lightFetcher) getMaxConfirmedTd() *big.Int {
```

Comment:
Getter functions that just return a field are not necessary if they are never called externally.